### DISCUSSION TITLE
How to set focus at the beginning of the document

### DISCUSSION THREAD

**Discussion by VBobCat** [sibiraj-s/ngx-editor#453] [Q&A] [answered]
My editor is bound to a formControl. 
Then I import an html document and set the formControl's value this way:
```
this.formControlDoc.setValue(toDoc(html));
```
The editor then shows the document correctly. However, when I put focus on the editor, its cursor is at the end of the document.
How can I set the cursor position to the beginning of the document?
Thank you very much.

**Comment by sibiraj-s** [comment, marked as answer]
You should be able to do

```js
editor.commands.focus(’start’).exec()
```

since v15.2.0

**Reply by VBobCat** [reply]
Wow, that was fast! Thank you very much!

[a comment by an automated account is not shown]
